utils/config.go

```diff
@@ -442,35 +442,42 @@ func (config *Config) evaluateCertificate() error {
 	return nil
 }
 
-// GetConfigFilePath returns concerto configuration path file
-func GetConfigFilePath(fileFilter string) string {
-	log.Debug("GetConfigFilePath")
-	path := ""
-	switch {
-	case fileFilter == "LogFile":
-		if runtime.GOOS == "windows" {
-			path = windowsServerLogFilePath
-		} else {
-			path = nixServerLogFilePath
-		}
-	case fileFilter == "CaCertPath":
-		if runtime.GOOS == "windows" {
-			path = windowsServerCaCertPath
-		} else {
-			path = nixServerCaCertPath
-		}
-	case fileFilter == "CertPath":
-		if runtime.GOOS == "windows" {
-			path = windowsServerCertPath
-		} else {
-			path = nixServerCertPath
-		}
-	case fileFilter == "KeyPath":
-		if runtime.GOOS == "windows" {
-			path = windowsServerKeyPath
-		} else {
-			path = nixServerKeyPath
-		}
+// GetDefaultLogFilePath returns default concerto configuration path file
+func GetDefaultLogFilePath() (path string) {
+	if runtime.GOOS == "windows" {
+		path = windowsServerLogFilePath
+	} else {
+		path = nixServerLogFilePath
+	}
+	return
```

Comment:
Using named return values is not considered very idiomatic Go. I would actually refactor this as follows:
```Go
func GetDefaultLogFilePath() (path string) {
  if runtime.GOOS == "windows" {
    return windowsServerLogFilePath
  }
  return nixServerLogFilePath
}
```